coxeter/families/common.py

```diff
@@ -26,7 +26,7 @@ class RegularNGonFamily(ShapeFamily):
 
     @classmethod
     def get_shape(cls, n):
-        """Generate an n-gon.
+        """Generate an n-gon with area 1.
```

Comment:
```suggestion
        """Generate a unit area n-gon.
```
My personal preference, but also what's already used elsewhere in the docstrings.